pymatviz/ptable.py

```diff
@@ -599,7 +599,9 @@ def add_colorbar(
         """
         # Update colorbar kwargs
         cbar_kwargs = {"orientation": "horizontal"} | (cbar_kwargs or {})
-        title_kwargs = {"fontsize": 12, "pad": 10} | (title_kwargs or {})
+        title_kwargs = {"fontsize": 12, "pad": 10, "label": title} | (
```

Comment:
oops, had `"label": title` in there initially but then accidentally pressed undo once too many 😅

Reply:
All good. Typos happen :)